var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -223,14 +223,19 @@ def cmake_args(self):
                     )
         else:
             args.append(self.define("DLAF_WITH_MKL", spec["lapack"].name in INTEL_MATH_LIBRARIES))
+            add_dlaf_prefix = lambda x: x if spec.version <= Version("0.6") else "DLAF_" + x
```

Comment:
just a more spack-pythonic way of saying the same thing.

```suggestion
            add_dlaf_prefix = lambda x: x if spec.satisfies(":0.6") else "DLAF_" + x
```